### ISSUE TITLE
run_text appears to be passed in as global

### ISSUE THREAD

**Issue by rburghol** [HARPgroup/hydro-tools#278]
@jdkleiner - I don't want to interrupt your coding by throwing a pull request in there, but I noticed the variable "run_text" is not defined inside `om_cia_table()`, nor is it passed in as an argument.  When you have a minute to make one of those modifications that would be awesome.

**Comment by jdkleiner** [comment]
@rburghol Yeah run_text() is actually a function in the hydrotools package that I made using your example code in test_store_config.R, so no need to pass in. Related -> I was able to get the om_cia_table() updated to include the dynamic labeling using that new methodology, but it significantly slows the script down, the conventional way of getting properties seems to work quicker for retrieving the scenario_name. Just something worth taking a look at next time you dig into om_cia_table()
